libraries/joomla/form/fields/civiperms.php

```diff
@@ -2,8 +2,12 @@
 
 defined('JPATH_PLATFORM') or die;
 
-// for some reason Joomla doesn't autoload JFormFieldRules in this context
-require_once JPATH_SITE . '/libraries/joomla/form/fields/rules.php';
+if (!version_compare(JVERSION, '4.0.0', 'ge')) {
```

Comment:
@monishdeb is there a reason not to do this instead because it is kind of like your say if not greater than this instead of if less than this 
```suggestion
if (version_compare(JVERSION, '4.0.0', 'lt')) {
```

Reply:
nope just copy paste code with `!` :) but yeah would be good to use that, I have (force)updated the PR with `file_exists` condition now.